content/docs/tutorials/interactivity.md

```diff
@@ -0,0 +1,26 @@
+---
+$title: Create interactive AMP pages
```

Comment:
Thanks.  I plan on correcting the other title as sentence case s/b used.  Hmm, I might change the title to "Create advanced interactive pages"